styles.css

```
@@ -433,19 +503,30 @@ svg {
   height: 4em;
 }
 
-.site-header__text:hover {
+.site-header__text,
+#link-to-top {
```

I think we still need to change this selector to match a class instead of an ID.

```suggestion
.link-to-top {
```